addon/webextension/selector/uicontrol.js

```diff
@@ -166,9 +175,18 @@ this.uicontrol = (function() {
       selectedPos = new Selection(
         0, 0,
         width, height);
-      shooter.takeShot("fullPage", selectedPos);
+      captureType = 'fullPage';
+      setState("previewing");
+    },
+    onSavePreview: () => {
+      sendEvent(`capture-preview`, "save-preview-button");
```

Comment:
These probably should include `captureType` as part of the event (maybe a label)